package.json

```diff
@@ -23,7 +23,7 @@
     "interactjs": "1.3.4",
     "lodash.clonedeep": "^4.5.0",
     "moment": "^2.24.0",
-    "mydatepicker": "^2.6.6",
+    "mydatepicker": "^9.0.2",
```

Comment:
Du müsstest das auch noch in der **lib/package.json** unter den `peerDependencies` anpassen